src/encoded/helpers/__init__.py

```diff
@@ -0,0 +1,5 @@
+"""
+Available helper functions, classes, and vars
```

Comment:
yea, we don't have to go in the helpers direction.  I like helpers/ or helpers.py because it can be used to temporarily store code that may eventually find it's own module.  But I like modules too!

I'll create a module for this and remove the helpers.